### PR TITLE
Ensure all deprecated bindings from AA are re-exported

### PR DESCRIPTION
**Pull request by fingolfin**
This is a step towards resolving https://github.com/oscar-system/Oscar.jl/pull/3114 (CC @thofma)

Requires https://github.com/Nemocas/AbstractAlgebra.jl/pull/1534 in a release of AA.

### CODE REVIEW

File: src/Aliases.jl [hunk 1]
```diff
@@ -1,3 +1,9 @@
+import AbstractAlgebra: LaurentMPolyRing  # FIXME: remove these imports once AA exports these
+import AbstractAlgebra: LaurentMPolyRingElem
+import AbstractAlgebra: LaurentPolyRing
+import AbstractAlgebra: LaurentPolyRingElem
```
Comment (fingolfin) [inline src/Aliases.jl:4]: This is just there to make us independent from AA changes.

**Comment by fingolfin** [comment]
I am now working on doing the same for Hecke (and then for Oscar), and noticed we don't export `is_embedded` and `ZZModRing`, which I think we should export (anyone agree/disagree?)

But also `is_cyclo_type` and `is_maxreal_type` are types which are in `src/Aliases.jl` but are not exported. Do we want to export them? @thofma , @fieker ?

**Comment by thofma** [comment]
No need to export them at this point.

**Comment by fingolfin** [comment]
Shall we merge then?

File: src/Aliases.jl (hunk 1 above)
Comment (fingolfin) [reply src/Aliases.jl:4]: I am going to make a new AA release shortly; afterwards I can remove the above

**Comment by thofma** [comment]
Shall we remove those things now that we have an AA release?

**Comment by fingolfin** [comment]
PR updated, with useless stuff removed

**Comment by thofma** [comment]
Is this supposed to work with Oscar#master or only with Oscar#mh/fingolfin? I am asking because of the downstream tests.

**Comment by fingolfin** [comment]
There is no branch `mh/fingolfin` AFAIK?

Anyway, this is supposed to work with `master`, i.e., it is not breaking.

**Comment by lgoettgens** [comment]
I think Tommy means fingolgin:mh/aliases

**Pull request merged**